### ISSUE TITLE
Link Operator Readme to the HotelReservation/Kubernetes tutorial

### ISSUE THREAD

**Issue by RS1999ent** [docc-lab/DeathStarBench#1]
Modify the section on Kubernetes operators in the Kubernetes tutorial readme to link to the Operator repo/branch.  Include a high-level description.   The idea is for the Kubernetes tutorial to be the entry point to learning about Kubernetes.

**Comment by RoyZhang7** [comment]
Just making sure the k8s tutorial you're referring to these two: [instructions inside profile repo](https://github.com/docc-lab/hacking-exercises/blob/main/Kubernetes-basic-tutorial.md) and [tutorial summary inside hacking-exercise repo](https://github.com/docc-lab/hacking-exercises/blob/main/Kubernetes-basic-tutorial.md)

**Comment by RS1999ent** [comment]
Yes.  Though both of your hyperlinks above point to the same URL :).   The hacking-exercise repo's kubernetes operator sectoin should simply point to the "instructoins inside profile repo of operator branch". Make sense?

**Comment by RoyZhang7** [comment]
Oops, the first link I was referring to [readme under operator dir](https://github.com/docc-lab/DeathStarBench/tree/operator/hotelReservation/k8s-operator/frontservices-operator)

**Comment by RoyZhang7** [comment]
It turns out I don't have permission to write the tutorial repo. @RS1999ent @MaxLanLiu Could you guys add me as a collaborator?
